entity-service/src/main/kotlin/com/egm/stellio/entity/web/EntityHandler.kt

```diff
@@ -22,7 +26,7 @@ import reactor.core.publisher.Mono
 import reactor.kotlin.core.publisher.toMono
 import java.net.URI
 
-@Component
+@Controller

```

Comment:
😨 😨 😨

Reply:
We can only use @RequestMapping on @Controller annotated class.
I saw also that the @Controller extends the @Component bean.